src/track-by-label-id/format-response.ts

```diff
@@ -1 +1,62 @@
-export {};
+import { Response } from "./types/private";
+import { TrackByLabelIdTypes } from ".";
+import {
+  TrackingEvent,
+  TrackingStatusCodes,
+  TrackingStatusDescription,
+} from "./types/public";
+
+export function formatResponse(
+  response: Response.GetTrackingLogFromLabelResponseBody
+): TrackByLabelIdTypes.Response {
+  return formatTrackByLabelIdResult(response);
+}
+
+function formatTrackByLabelIdResult(
+  result: Response.GetTrackingLogFromLabelResponseBody
+): TrackByLabelIdTypes.TrackByLabelIdResult {
+  return {
+    trackingNumber: result.tracking_number || "",
```

Comment:
are we moving away from the pattern of making all results non optional?

Reply:
@j-r-j I had the same question on your `createLabel` PR. I'd like to sync up on this with you and Pierce after standup.